widget/ui/src/components/Toast/Toast.tsx

```diff
@@ -14,23 +14,26 @@ import {
   ToastContentContainer,
 } from './Toast.styles';
 
-const DEFAULT_HIDE_DURATION = 3_000;
-
 export const Toast = (props: ToastProps) => {
-  const { id, horizontal, autoHideDuration = DEFAULT_HIDE_DURATION } = props;
+  const { id, autoHideDuration, onClose } = props;
   const { removeToast } = useToast();
   useEffect(() => {
-    const timer = setTimeout(() => {
-      removeToast(id);
-    }, autoHideDuration);
+    let cleanup;
+
+    if (autoHideDuration) {
```

Comment:
Please check animation / close problem in dapp.

Reply:
I think the animation should be handled in another task. But I fixed the overlap issue.